src/core/shape_inference/include/convolution_shape_inference.hpp

```diff
@@ -51,5 +54,8 @@ std::vector<TRShape> shape_infer(const TOp* op,
     return output_shapes;
 }
 }  // namespace v1
+
+using v1::shape_infer;
```

Comment:
Could `using` be avoided in .hpp file? How complex is alternative solution?
fyi @praasz

Reply:
It should be removed or have some local scope only like: function, code block etc.